api/src/index.ts

```diff
@@ -11,6 +11,7 @@ nconf
       port: 5432,
       schema: "public",
     },
+    adminToken: "local"
```

Comment:
With it being initialized this way, does this mean that if the prod config somehow loses the admin token entry, the public site will fall back to `local`? It might be better to have those APIs more or less disabled unless the admin token is configured in the secrets file, just to be safe.

Reply:
That's a good point. The local story gets a little more gnarly then but I guess we can put it in part of the 1 time setup that people set the value in their config.

Reply:
Updated